src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -450,14 +474,41 @@ public String getQuestionResultStatisticsCsv(
 
         DecimalFormat df = new DecimalFormat("#.##");
         
+        Map<String, List<Integer>> sortedOptionPoints = new TreeMap<String, List<Integer>>();
+        
+        Map<String, String> identifierMap = new HashMap<String, String>();
+        
         for (Entry<String, List<Integer>> entry : optionPoints.entrySet()) {
+            
+            if (distributeToRecipients) {
+                
+                String participantIdentifier = entry.getKey();
+                String name = bundle.getNameForEmail(participantIdentifier);
+                String nameEmail = name + participantIdentifier;
+                
+                identifierMap.put(nameEmail, participantIdentifier);
+                sortedOptionPoints.put(nameEmail, entry.getValue());
+                
+            } else {
+                
+                String option = options.get(Integer.parseInt(entry.getKey()));
+                
+                sortedOptionPoints.put(option, entry.getValue());
+            }
+        }
```

Comment:
That depends on the nature of the function extracted. If it is specific to this class only, we should keep it in this class itself. If it is reusable by other classes, we can move it to a more suitable place.